grafana/dashboards/dashboard-results.json

```diff
@@ -3294,7 +3294,7 @@
                 "type": "prometheus",
                 "uid": "PEDF09670F151C95C"
               },
-              "expr": "k6_my_counter{testid=~\"$testid\"}",
+              "expr": "k6_my_counter_total{testid=~\"$testid\"}",
```

Comment:
It depends on what the objective of this dashboard is, if it is to show test results it may not be necessary, but if it is an example for others to build their own dashboard it could make the job easy.

I have no problem removing it.

Reply:
@ppcano @javaducky can you share your thoughts here, please?

Let's merge for now. We will fix it in the next few days if needed.